CMakeLists.txt

```diff
@@ -109,14 +109,10 @@ execute_process(
   OUTPUT_VARIABLE GIT_BRANCH
   OUTPUT_STRIP_TRAILING_WHITESPACE
 )
-configure_file(templates/setup.sh.in ${CMAKE_CURRENT_BINARY_DIR}/setup.sh @ONLY)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/setup.sh
+configure_file(templates/thisepic.sh.in ${CMAKE_CURRENT_BINARY_DIR}/thisepic.sh @ONLY)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/thisepic.sh
     DESTINATION ${CMAKE_INSTALL_PREFIX}
 )
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/setup.sh
```

Comment:
No deprecation period?

Reply:
Nightly is bleeding edge?

Reply:
Deprecation is when we make a release that nudges users to update their scripts (Ideally, for a prolonged period of time). Removing the file in nightly is a rug pull.

Reply:
```
$ source setup.sh 
Warning: This script will cease to exist at '/home/wdconinc/git/epic/prefix/setup.sh'.
         Please use the version at '/home/wdconinc/git/epic/prefix/bin/thisepic.sh'.
```

Reply:
I'm a bit worried that this will turn into:
```
$ source setup.sh 
Warning: This script will cease to exist at '/home/wdconinc/git/epic/prefix/setup.sh'.
         Please use the version at '/opt/software/linux-debian12-x86_64_v2/gcc-12.2.0/epic-24.05.0-dqdgda7hmogjwe5qimc4s2vlut3ii4fh/bin/thisepic.sh'.
```

Reply:
Not sure how to avoid this by only going through one level of symlinking.